source/style-guide/information-types.txt

```diff
@@ -0,0 +1,75 @@
+.. _information-types:
+
+=================
```

Comment:
Suggestion:
I think it could be separate this information typing page from the prototypes by creating a separate directory for them and adding them to an "information-types" drawer as shown below:

```
information-types/
├── index.txt (this page)
├── prototypes/
│   ├── concept.txt
│   ├── reference.txt
│   ├── task
```

In the current structure, you might expect the "concept.txt" page to explain the type more in depth rather than show a template.

Reply:
I can make this update. Is the suggestion here to only update the file structure, or to update the rendered ToC as well?

Reply:
Sorry, I forgot the prototypes.txt or index.txt within the prototypes directory in my example directory structure.
I think the ToC update would be great because it helps divide the conceptual information from the examples.

Reply:
No problem! I updated the ToC to nest the prototype pages under a top-level Prototype page.